### DISCUSSION TITLE
Firmware build for pineriver h24 mini-x 1gb

### DISCUSSION THREAD

**Discussion by Gokuh1980** [johang/sd-card-images#74] [General]
Hello, I need to compile firmware with Debian for a Minix Pineriver H24 with an Allwinner A10. I see the instructions to build it, but they are too complex for me and I'm afraid of making mistakes. When comparing the FEL files from the MK802 and the 1GB MINIX, I notice some differences in the hardware description. Can someone help me resolve this or compile it for me? On the other hand, I understand that the firmware will go on a micro SD card and not on the 4GB internal NAND of the SBC. Is this correct? Is there a method to install the firmware directly into the internal memory? Thank you very much.

**Comment by johang** [comment]
I added this board. It will show up on the site after the next rebuild (September 1).

**Reply by Gokuh1980** [reply]
Wow, Thanks!!! It´s easier to flash with balena etcher to a micro sd and boot from there or to burn it in the nand flash?

**Reply by johang** [reply]
Don't know.
